pallets/summary/src/lib.rs

```diff
@@ -127,8 +127,12 @@ pub mod pallet {
 
         /// Weight information for the extrinsics in this pallet.
         type WeightInfo: WeightInfo;
-
+        /// An Ethereum bridge provider
         type BridgeInterface: avn::BridgeInterface;
+        /// A flag to determine if summaries will be automatically sent to Ethereum
+        type AutoSubmitSummaries: Get<bool>;
+        /// A unique instance id to differentiate different instances
+        type InstanceId: Get<u8>;
```

Comment:
The `AutoSubmitSummaries` and `InstanceId` could be integrated into the `BridgeInterface` Trait, making it more explicit.

Reply:
`BridgeInterface` shouldn't need to know the business logic of its calling pallets. It responsibility is to either send a transaction to Ethereum or generate a lower proof.